### PR TITLE
Add a coupled code card for deal.II

### LINKED ISSUE #49: Add deal-ii adapter on the website
Recent addition to our family: https://github.com/precice/dealii-adapter

### PR DESCRIPTION
**Pull request by davidscn**
Closes #49.

I have not viewed at the website locally, so please check the result before merging.

### CODE REVIEW

**Comment by shkodm** [comment]
I've just checked locally. Looks good.

File: _codes/dealii.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+status: up-to-date
+application: Finite element method, in this case fluid-structure interaction (solid part)
+can-write: Displacement
+can-read: Forces
+notes: Linear elastic soid mechanics 
+---
+
+[deal.II](https://www.dealii.org/) is an open source software library supporting the creation of finite element codes. It uses state-of-the-art programming techniques to offer a modern interface to the complex data structures and algorithms required.
+
+Build status:<a style="text-decoration: none" href="https://travis-ci.org/precice/dealii-adapter" target="_blank"><img src="https://travis-ci.org/precice/dealii-adapter.svg?branch=master" alt="Build status">
```
Comment (shkodm) [inline _codes/dealii.md]: Add one space after build status, before travis status. Otherwise the margin is too small.

**Review by shkodm — APPROVED** [review]

**Comment by shkodm** [comment]
Also, should we add some tutorial with deal.ii to the `resources` section?

File: _codes/dealii.md (hunk 1 above)
Comment (davidscn) [reply _codes/dealii.md]: Hope it's better now.

**Comment by davidscn** [comment]
> Also, should we add some tutorial with deal.ii to the `resources` section?

Good question, maybe @MakisH knows. is this related to #45 ?

**Review by MakisH — CHANGES_REQUESTED** [review]
This builds nicely (it's very easy to try it locally, have a look at the README).

The information looks good. However, please:
- [x] Add the supported deal-II versions (even if you only know one)
- [x] Move the card up (e.g. before or after FEniCS)

Regarding the [Resources](https://www.precice.org/resources/) page: The page is not currently designed to point to different variants of each tutorial (or, let's say, many combinations that we have documented are missing). In terms of physics/geometry we don't need to add anything. If you want to add anything, you could use a different screenshot. In any case, this is a different issue, related to https://github.com/precice/tutorials/issues/28.

**Comment by davidscn** [comment]
Concerning the version: Most of  the used functions are quiet basic. So, I cannot really say, which deal.II version are going to work and which not. Of course, I could mention there the most recent version the adapter has been built on. But this is probably a restriction.

**Review by MakisH — APPROVED** [review]

**Comment by MakisH** [comment]
> Concerning the version: Most of the used functions are quiet basic. So, I cannot really say, which deal.II version are going to work and which not. Of course, I could mention there the most recent version the adapter has been built on. But this is probably a restriction.

Still, giving one specific "known-to-work" version to the user is invaluable. I fixed the field and added more information. LGTM.

**Pull request merged**